src/log-store/src/kafka/periodic_offset_fetcher.rs

```diff
@@ -12,98 +12,91 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::sync::Arc;
 use std::time::Duration;
 
-use common_telemetry::error;
-use dashmap::DashMap;
-use store_api::logstore::provider::KafkaProvider;
+use common_telemetry::{debug, error, info};
 use tokio::time::{interval, MissedTickBehavior};
 
 use crate::error::Result;
 use crate::kafka::client_manager::ClientManagerRef;
 
-/// HighWatermarkManager is responsible for periodically updating the high watermark
-/// (latest existing record offset) for each Kafka topic.
-pub(crate) struct HighWatermarkManager {
-    /// Interval to update high watermark.
-    update_interval: Duration,
-    /// The high watermark for each topic.
-    high_watermark: Arc<DashMap<Arc<KafkaProvider>, u64>>,
+/// PeriodicOffsetFetcher is responsible for periodically updating the offset
+/// for each Kafka topic.
+pub(crate) struct PeriodicOffsetFetcher {
+    /// Interval to fetch the latest offset.
+    interval: Duration,
     /// Client manager to send requests.
     client_manager: ClientManagerRef,
 }
 
-impl HighWatermarkManager {
-    pub(crate) fn new(
-        update_interval: Duration,
-        high_watermark: Arc<DashMap<Arc<KafkaProvider>, u64>>,
-        client_manager: ClientManagerRef,
-    ) -> Self {
+impl PeriodicOffsetFetcher {
+    pub(crate) fn new(update_interval: Duration, client_manager: ClientManagerRef) -> Self {
         Self {
-            update_interval,
-            high_watermark,
+            interval: update_interval,
             client_manager,
         }
     }
 
-    /// Starts the high watermark manager as a background task
+    /// Starts the offset fetcher as a background task
     ///
     /// This spawns a task that periodically queries Kafka for the latest
-    /// high watermark values for all registered topics and updates the shared map.
+    /// offset values for all registered topics and updates the shared map.
     pub(crate) async fn run(self) {
+        info!("PeriodicOffsetFetcher started");
         common_runtime::spawn_global(async move {
-            let mut interval = interval(self.update_interval);
+            let mut interval = interval(self.interval);
```

Comment:
```suggestion
        common_runtime::spawn_global(async move {
            info!("PeriodicOffsetFetcher started");
            let mut interval = interval(self.interval);
```

Reply:
resolved in [3524959](https://github.com/GreptimeTeam/greptimedb/pull/6730/commits/3524959069e09beee6bbe9fd6e259f880779b18d)